README.md

```diff
@@ -1,20 +1,72 @@
-Siddhi-gpl-execution-pmml
+siddhi-gpl-execution-pmml
 ======================================
 
-This is an extension for processing texts based on a PMML processing model. 
+The **siddhi-gpl-execution-pmml extension** is an extension to <a target="_blank" href="https://wso2.github.io/siddhi">Siddhi
+</a> that  can be used to process texts based on a PMML processing model.
 
-#### Deploying the feature
- Feature can be deployed as an OSGI bundle by putting the jar file of the component to SP_HOME/lib directory of SP 4.0.0 pack. 
+Find some useful links below:
 
-## How to Contribute
-* Please report issues at [Siddhi Github Issue Tacker](https://github.com/wso2-extensions/siddhi-gpl-execution-pmml)
-* Send your bug fixes pull requests to [master branch](https://github.com/wso2-extensions/siddhi-gpl-execution-pmml/tree/master) 
+* <a target="_blank" href="https://github.com/wso2-extensions/siddhi-gpl-execution-pmml">Source code</a>
+* <a target="_blank" href="https://github.com/wso2-extensions/siddhi-gpl-execution-pmml/releases">Releases</a>
+* <a target="_blank" href="https://github.com/wso2-extensions/siddhi-gpl-execution-pmml/issues">Issue tracker</a>
+
+## Latest API Docs 
+
+Latest API Docs is <a target="_blank" href="https://wso2-extensions.github.io/siddhi-gpl-execution-pmml/api/4.0.2-SNAPSHOT">4.0.2-SNAPSHOT</a>.
+
+## How to use 
+
+**Using the extension in <a target="_blank" href="https://github.com/wso2/product-sp">WSO2 Stream Processor</a>**
+
+* You can use this extension in the latest <a target="_blank" href="https://github.com/wso2/product-sp/releases">WSO2 Stream Processor</a> that is a part of <a target="_blank" href="http://wso2.com/analytics?utm_source=gitanalytics&utm_campaign=gitanalytics_Jul17">WSO2 Analytics</a> offering, with editor, debugger and simulation support. 
+
+* This extension is shipped by default with WSO2 Stream Processor, if you wish to use an alternative version of this extension you can replace the component <a target="_blank" href="https://github.com/wso2-extensions/siddhi-gpl-execution-pmml/releases">jar</a> that can be found in the `<STREAM_PROCESSOR_HOME>/lib` directory.
```

Comment:
We are not shipping gpl extension with SP right?

Reply:
Yes so please modify accordingly.

Reply:
Fixed in de755f9